demos/mixer-config-quota-bookinfo.yaml

```diff
@@ -22,6 +22,50 @@ data:
         params:
       - name: default
         impl: denyChecker
+    attributes:
```

Comment:
Yup, but right now we don't have a way for the proxy to provide attributes. So today we encode it statically in our global config, with the plan to replace the "static" with the proxy pushing a manifest.